projects/plugins/jetpack/extensions/blocks/blogging-prompt/index.js

```diff
@@ -19,8 +19,9 @@ export const settings = {
 	description: (
 		<Fragment>
 			<p>{ __( 'Answer a new and inspiring writing prompt each day.', 'jetpack' ) }</p>
-			{ /* @TODO add link */ }
-			<ExternalLink href="#">{ __( 'Learn more.', 'jetpack' ) }</ExternalLink>
+			<ExternalLink href="https://support.wordpress.com/wordpress-editor/blocks/writing-prompt-block/">
+				{ __( 'Learn more.', 'jetpack' ) }
+			</ExternalLink>
```

Comment:
I'm afraid we'll need to remove the link entirely now. See #26792 for more info.

Reply:
Ah, good to know. I'll remove the link.